docs/artifact-migrations.md

```diff
@@ -31,6 +31,6 @@ to be greater than the latest version with the old groupId / old artifact id.
 
 Pull requests that added artifact migrations can be found [here][migration-prs].
 
-[migrations]: https://github.com/scala-steward-org/scala-steward/blob/master/modules/core/src/main/resources/artifact-migrations.conf
+[migrations]: https://github.com/scala-steward-org/scala-steward/blob/master/modules/core/src/main/resources/artifact-migrations.v2.conf
```

Comment:
In this file are still reference to `initialVersion`. I had forgotten to remove them I guess.

Reply:
Good point. I removed the reference in the latest commit.